.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Add this to your `.gitignore` file

Reply:
It is I added the .gitignore it to the commit

Reply:
Oh oh. LOOOOOOL. Sorry

I didn't see the file name 😭